app/src/main/java/ch/blinkenlights/android/vanilla/PreferencesActivity.java

```diff
@@ -101,6 +100,16 @@ public void onSharedPreferenceChanged (SharedPreferences sharedPreferences, Stri
 		}
 	}
 
+	@Override
+	public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
This is the fix for the back button, right?

Why do we need the onStart overrides?

Reply:
That is related to the old problem with the system back button (the triangle at the bottom). I found the past fixes only work for O but not below N.